Assets/Scripts/InstructionsButton.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InstructionsButton : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	public void changeScene(string sceneName)
+    {
+        Application.LoadLevel(sceneName);

```

Comment:
`Application.LoadLevel` is [obsolete](https://docs.unity3d.com/ScriptReference/Application.LoadLevel.html), you should use `SceneManager` instead.

[Assets/Onboarding/ExitOnboarding.cs](https://github.com/ubcemergingmedialab/Soil-topARgraphy/blob/master/Assets/Onboarding/ExitOnboarding.cs) has an example on how to load a scene.

Reply:
⬆️